src/Microsoft.Performance.SDK/Processing/ColumnBuilding/IColumnBuilder.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Performance.SDK.Processing.ColumnBuilding;
+
+/// <summary>
+///     Base interface for classes that can build/extend upon columns that are
+///     part of a table.
+/// </summary>
+public interface IColumnBuilder
```

Comment:
>IColumnBuilder [](http://example.com/codeflow?start=17&length=14)

Should we make this name more specific, maybe by adding Variant?

Reply:
I would prefer not to in case we ever want to expose more operations on columns